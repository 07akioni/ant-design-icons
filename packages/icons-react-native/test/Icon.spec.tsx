import * as React from 'react';
import * as renderer from 'react-test-renderer';

import IconFill, { fillGlyphMap, IconFillProps } from '../src/fill';

function mount(props: IconFillProps) {
  const component = renderer.create(<IconFill {...props} />);
  return component.toJSON();
}

describe('IconFill', () => {
  it('should create Icon element.', () => {
    const icon = mount({ name: 'account-book' });
    expect(icon!.children![0]).toBe(String.fromCharCode(fillGlyphMap['account-book']));
    expect(icon).toMatchSnapshot();
  });
});
