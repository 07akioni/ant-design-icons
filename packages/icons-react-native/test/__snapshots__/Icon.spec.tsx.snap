// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconFill should create Icon element. 1`] = `
<
  style={
    Array [
      Object {
        "color": "black",
        "fontFamily": "antfill",
        "fontSize": 14,
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      undefined,
    ]
  }
>
  
</>
`;
